esm-samples/jsapi-esm-cdn/README.md

```diff
@@ -1,6 +1,6 @@
 # ArcGIS Maps SDK for JavaScript ES modules via CDN
 
-We currently recommend only using the ES modules (ESM) via CDN for testing and prototyping. The ESM CDN build isn't optimized for fast loading. 
+We currently recommend only using the ES modules (ESM) via CDN for testing and prototyping. The ESM CDN hasn't been built, and is not optimized for fast loading. 
```

Comment:
I'm not sure what is meant my "hasn't been built"?

Reply:
Hmm, I'm not sure now, either, lol. I'll revert.

Reply:
Reverted.